ILRepack/RepackOptions.cs

```diff
@@ -79,8 +78,10 @@ public List<string> AllowedDuplicateNameSpaces
         private readonly RepackAssemblyResolver globalAssemblyResolver = new RepackAssemblyResolver();
         private List<Regex> excludeInternalizeMatches;
 
-        public void SetSearchDirectories(string[] dirs)
+        public void SetSearchDirectories(IEnumerable<string> dirs)
         {
+            if (TargetPlatformDirectory != null)
+                dirs = new[] { TargetPlatformDirectory }.Concat(dirs);
```

Comment:
One crazy idea would be to do something that C-based configuration tools already do: compile a simple C# program and maybe try to gather from that the info (see where the referenced dlls are). I'll take a look into that maybe.